packages/frontend/scss/misc/_context_menu.scss

```
@@ -41,6 +41,15 @@
     0 2px 4px rgba(16, 22, 26, 0.2),
     0 8px 24px rgba(16, 22, 26, 0.2);
 
+  hr.seperator {
+    background-color: #8080804a;
+    height: 2px;
+    border: 0;
+    margin: 2px;
```

let's make the things a bit more subtle, as in many other apps and os - thinner line, more whitespace:

 ```suggestion
     height: 1px;
     border: 0;
     margin: 6px;
```